app/controllers/api/v1/trips_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::TripsController < ApplicationController
+
+  def create
+    trip = Trip.create(name: params["destination"],
+      coord_lat: params["coord_lat"],
+      coord_lng: params["coord_lng"])
```

Comment:
you can use symbols here rather than strings

Reply:
yes! prefer them symbols too